pyccel/codegen/printing/pycode.py

```diff
@@ -360,6 +360,9 @@ def _print_Break(self, expr):
     def _print_Continue(self, expr):
         return 'continue'
 
+    def _print_NoneType(self, expr):
+        return 'None'
```

Comment:
We don't want to print `None`. If `None` was passed to the function then it will be a `Nil` object by this stage, if nothing was passed then we will have `None` at this stage, this just means that we don't want to print the argument. Otherwise we will have unreadable code. E.g.:
```
x = full((3,4), 4.5, dtype=None, order=None)
```
This situation will get worse and worse for more complex functions with lots of arguments